src/App.js

```diff
@@ -2,10 +2,13 @@ import './App.css'
 import { BrowserRouter } from 'react-router-dom'
 import { Header, Footer } from './components'
 import { Router } from './router/Router'
+import { ToastContainer } from 'react-toastify'
+import 'react-toastify/dist/ReactToastify.css'
```

Comment:
why importing this twice, once in app.css and this time in app.js

Reply:
Thanks for pointing out this.